submodules/usersCallForwarding/views/complexStrategy.html

```diff
@@ -0,0 +1,184 @@
+<div class="complex-strategy-content" data-strategy="{{strategy}}">
+    <div class="complex-strategy-header">
+        <div class="forward-from">
+            <label>
+                Forward calls from
+            </label>
+
+            <div class="complex-rule">
+                {{#monsterRadio @root.i18n.users.callForwarding.strategies.selective.all }}
+                <input type="radio" class="radio-state" name="{{strategy}}.from" value="allNumbers" checked="checked"
+                    {{#compare type '===' 'allNumbers' }} checked{{/compare}} />
+                {{/monsterRadio}}
+
+                <div>
+                    {{#monsterRadio @root.i18n.users.callForwarding.strategies.selective.specific }}
```

Comment:
same here and in few others. You would only need inside loops
```suggestion
                    {{#monsterRadio i18n.users.callForwarding.strategies.selective.specific }}
```